charts/spire/charts/spire-server/values.yaml

```diff
@@ -184,6 +183,25 @@ upstreamAuthority:
         certificate: ""
         key: ""
         bundle: ""
+  awsPCA:
+    enabled: true
+    # -- AWS Region to use
+    region: "us-west-2"
+    # -- ARN of the "upstream" CA certificate
+    certificateAuthorityArn: "arn:aws:acm-pca:us-west-2:945673678728:certificate-authority/636dbcd8-203f-43e0-9701-be854b094a2b"
+    # -- (Optional) ARN of an IAM role to assume
+    assumeRoleArn: "arn:aws:iam::945673678728:role/PCAAccessSpireRole"
```

Comment:
Guessing you probably don't want to be pointing everyone at your own things by default. Leaked in by mistake during testing?

Reply:
my test file got in a way - appologies.